static/docs/commands-reference/move.md

```diff
@@ -106,4 +106,34 @@ To track the changes with git run:
 
 	git add .gitignore .gitignore data.csv.dvc  
 ```
+Here we use `dvc add` to put a file under DVC control. Then we use `dvc move` to change data directory. Note, that data stage file is also moved. If target path already exists and is a directory, data file is moved with unchanged name into this folder.
+
+```
```

Comment:
put `dvc` right after triple accent sign to enable syntax highlight

Reply:
dvc or text?

Reply:
`dvc` - we have a custom highlighter. Check the `run.md` fo the reference.